Gordon360/Models/ViewModels/RecIM/ParticipantExtendedViewModel.cs

```diff
@@ -14,6 +14,8 @@ public class ParticipantExtendedViewModel
         public string Status { get; set; }
         public IEnumerable<ParticipantNotificationViewModel> Notification { get; set; } 
         public bool IsAdmin { get; set; }
+        public bool AllowEmails { get; set; }
+        public string SpecifiedGender { get; set; }
```

Comment:
Specified is to encapsulate the users who are `not` Gordon students in which is a case where we need to talk to our customer about how we want to assign that. However, it will still be a `specified` gender since it can be modified.

Reply:
non-blocking